core/string/ustring.cpp

```diff
@@ -939,6 +939,7 @@ const char32_t *String::get_data() const {
 }
 
 void String::erase(int p_pos, int p_chars) {
+	p_pos = MAX(p_pos, 0);
 	*this = left(p_pos) + substr(p_pos + p_chars, length() - ((p_pos + p_chars)));
```

Comment:
```suggestion
	*this = left(MAX(p_pos, 0)) + substr(p_pos + p_chars, length() - ((p_pos + p_chars)));
```
Since only `left()` was changed shouldn't it be like this to restore previous behavior?